recipes/filesendercli/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "FileSenderCLI" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/madsi1m/FileSenderCLI/releases/download/{{ version }}/{{ name }}.tar.gz
+  sha256: 2dcabb8007769d2488e6c7055d49e273a1496d1b9811b67a0a3e361f2c97a308
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - filesender = filesender.filesender:cmd
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools >58.2.0
+    - wheel
+    - requests
```

Comment:
`requests`, `urllib3` and `cryptography` probably don't need to be in `host` since they aren't used by the `setup.py`. Sorry if I missed that previously.